src/NuGet/Microsoft.Orleans.Core.nuspec

```diff
@@ -24,6 +24,5 @@
   </metadata>
   <files>
     <file src="Orleans.dll" target="lib\net45" />
-    <file src="Orleans.xml" target="lib\net45" />
```

Comment:
I never quite know how useful the xml codedocs are without the .pdb files too, and/or whether the xml code-docs are supposed to be in the "symbol package" or here?

http://docs.nuget.org/Create/Creating-and-Publishing-a-Symbol-Package